server/src/main/java/org/elasticsearch/monitor/jvm/HotThreads.java

```diff
@@ -267,7 +268,8 @@ private String innerDetect() throws Exception {
                             sb.append(String.format(Locale.ROOT, "    %s%n", show[l]));
                         }
                     } else {
-                        sb.append(String.format(Locale.ROOT, "  %d/%d snapshots sharing following %d elements%n", count, threadElementsSnapshotCount, maxSim));
+                        sb.append(String.format(Locale.ROOT,
+                            "  %d/%d snapshots sharing following %d elements%n", count, threadElementsSnapshotCount, maxSim));
```

Comment:
I think it would be more consistent keeping the message string on the line above and only moving the parameters to this line.

Reply:
Thanks addressed in https://github.com/elastic/elasticsearch/pull/37456/commits/12d1de27d613d4c09c98874c2095aa2f43786925